maquetado/scss/footer.scss

```diff
@@ -0,0 +1,25 @@
+.footer-project {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  padding: 2rem 0;
+  position: relative;
+  width: 100%;
+}
+
+.footer-title,
+.footer-copyright {
+  color: $white;
+}
+
+.footer-title {
+  font-size: $text-big;
+}
+
+.footer-copyright {
+  bottom: 0;
+  font-size: $text-small;
+  position: absolute;
+  right: 10px;
```

Comment:
i think you could stop using `position: absolute and relative` and use flex for this, what do you think if you use `align-self: end` 👍

Reply:
I had tried but without result 🙄, now it works. 😅